storage/sql/migrate.go

```diff
@@ -100,7 +100,10 @@ var migrations = []migration{
 				connector_id text not null,
 				connector_data bytea,
 		
-				expiry timestamptz not null
+				expiry timestamptz not null,
+
+				code_challenge text,
+				code_challenge_method text
```

Comment:
This will not work for existing databases, I think. Adding to the table needs to be another migration altogether, I suppose...

Reply:
yes you are right, I will add another statement